File: src/main/java/hello/Repository/UserRepository.java
package hello.Repository;

import hello.Builder.UserBuilder;
import hello.User;
import java.util.ArrayList;
import java.util.List;

public class UserRepository {

    List<User> users = new ArrayList<User>();

    public UserRepository(){
        this.users.add(new UserBuilder().id(1).name("Yalçın Ceylan").age(27).build());
        this.users.add(new UserBuilder().id(2).name("Babak Partovi").age(37).build());
        this.users.add(new UserBuilder().id(3).name("Buğra Aslan").age(23).build());
    }

    public List<User> getUsers() {
        return users;
    }
}

File: src/main/java/hello/Repository/UserRepository.kt
package hello.Repository

import hello.Builder.UserBuilder
import hello.User
import java.util.*

class UserRepository {
    var users: MutableList<User> = ArrayList()
    init {
        users.add(UserBuilder().id(1).name("Yalçın Ceylan").age(27).build())
        users.add(UserBuilder().id(2).name("Babak Partovi").age(37).build())
        users.add(UserBuilder().id(3).name("Buğra Aslan").age(23).build())
    }
}